dns/types/entry.go

```diff
@@ -0,0 +1,36 @@
+package types
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// Entry is a value in the DNS with owners and content.
+type Entry struct {
+	// Owners represent the addresses which have authority over this entry.
+	Owners [][]byte `json:"owners" yaml:"owners"`
+
+	// Content represents the data of the entry.
+	Content []byte `json:"content" yaml:"content"`
```

Comment:
THis isn't the Content itself but an `IPLD` hash of the content

https://ipld.io/